cpp/include/cudf/types.hpp

```diff
@@ -18,8 +18,10 @@
 
 #ifdef __CUDACC__
 #define CUDF_HOST_DEVICE __host__ __device__
+#define CUDF_KERNEL      __global__ static
```

Comment:
Alignment not needed.
```suggestion
#define CUDF_KERNEL __global__ static
```